runsc/container/container_test.go

```diff
@@ -1843,8 +1843,8 @@ func TestUserLog(t *testing.T) {
 		t.Fatal("error finding test_app:", err)
 	}
 
-	// sched_rr_get_interval = 148 - not implemented in gvisor.
-	spec := testutil.NewSpecWithArgs(app, "syscall", "--syscall=148")
+        // sched_rr_get_interval - not implemented in gvisor.
+        spec := testutil.NewSpecWithArgs(app, "syscall", "--syscall="+sched_rr_get_interval_sysno)
```

Comment:
can we use syscall.SYS_SCHED_RR_GET_INTERVAL?
https://golang.org/pkg/syscall/

Reply:
> can we use syscall.SYS_SCHED_RR_GET_INTERVAL?
> https://golang.org/pkg/syscall/

Thanks, I will give it a try.

Reply:
> can we use syscall.SYS_SCHED_RR_GET_INTERVAL?
> https://golang.org/pkg/syscall/

It is a way better solution, thanks Andrei, patch updated.